model/task/task.go

```diff
@@ -55,8 +56,9 @@ var (
 )
 
 type Task struct {
-	Id     string `bson:"_id" json:"id"`
-	Secret string `bson:"secret" json:"secret"`
+	Id     string                       `bson:"_id" json:"id"`
+	Secret string                       `bson:"secret" json:"secret"`
+	Ann    []annotations.TaskAnnotation `bson:"annotation_doc" json:"annotation_doc"`
```

Comment:
A couple of things here:
1) It's best to name the variable something similar to what it actually is. Ann is fine since you know what you're looking at but it isn't really a consistent variable name, better to just say "Annotations".
2) This is a list of annotation documents, so "annotation_docs" or "annotations" is more correct.
3) I think this is only used for the resolver pipeline right? So can we move this to the bottom and add a comment indicating what it's for?

Reply:
Oops, I put that there for testing, I meant to take that out!

Reply:
Actually is this a holdover from testing? Do you need the annotation document in the task struct since you do the known issue casing within the same aggregation?

Reply:
Oops race :joy:

Reply:
done. Thanks for spotting it!